covid_plot.py

```diff
@@ -272,9 +272,9 @@ def plot_area(df: pd.DataFrame,
             df_plot.plot(ax=a1, y=perccols, kind='area', legend=False)
             a1.set_ylabel('Percent')
             a1.xaxis.label.set_visible(False)
-            a1.secondary_yaxis('right', functions=( lambda x: x, lambda x: x ))
+            a1.secondary_yaxis('right', functions=(lambda x: x, lambda x: x))
 
-        a0_secax_y = a0.secondary_yaxis('right', functions=( lambda x: x, lambda x: x ))
```

Comment:
Perfect, thanks! I approved the workflow run now that our regularly scheduled one finished. Don't worry if yours fails, it will be because of an issue we're working on right now but I just want to follow GH procedures and be sure there's no new problems introduced (can't imagine how!). :smile: